src/modules/person/api/person.controller.spec.ts

```diff
@@ -65,14 +73,39 @@ describe('PersonController', () => {
                     provide: UsernameGeneratorService,
                     useValue: createMock<UsernameGeneratorService>(),
                 },
+                {
+                    provide: PersonUc,
+                    useValue: createMock<PersonUc>(),
+                },
+                {
+                    provide: ConfigService,
+                    useValue: createMock<ConfigService>(),
+                },
             ],
         }).compile();
         personController = module.get(PersonController);
         personenkontextUcMock = module.get(PersonenkontextUc);
         personRepositoryMock = module.get(PersonRepository);
         usernameGeneratorService = module.get(UsernameGeneratorService);
+        personUcMock = module.get(PersonUc);
+        //configServiceMock = module.get(ConfigService);
+        personPermissionsMock = createMock<PersonPermissions>();
```

Comment:
I wouldn't create this mock for all tests. Instead create the mock in every test if it is needed. This is basically global state without the need to be